opba-db/src/main/resources/migration/migrations/0000-create-table-banks.xml

```diff
@@ -75,14 +78,16 @@
                 file="banks.csv"
                 catalogName=""
                 tableName="${table-prefix}bank_profile">
-            <column header="uuid"       name="bank_uuid"                type="STRING"/>
-            <column header="name"       name="name"                     type="skip"/>
-            <column header="bic"        name="bic"                      type="skip"/>
-            <column header="url"        name="url"                      type="STRING"/>
-            <column header="adapter_id" name="adapter_id"               type="STRING"/>
-            <column header="bank_code"  name="bank_code"                type="skip"/>
-            <column header="idp_url"    name="idp_url"                  type="STRING"/>
-            <column header="aspsp_sca_approaches" name="sca_approaches" type="STRING"/>
+            <column header="uuid"                   name="bank_uuid"                type="STRING"/>
+            <column header="name"                   name="name"                     type="skip"/>
+            <column header="bic"                    name="bic"                      type="skip"/>
+            <column header="url"                    name="url"                      type="STRING"/>
+            <column header="adapter_id"             name="adapter_id"               type="STRING"/>
+            <column header="bank_code"              name="bank_code"                type="skip"/>
+            <column header="idp_url"                name="idp_url"                  type="STRING"/>
+            <column header="aspsp_sca_approaches"   name="sca_approaches"           type="STRING"/>
+            <column header="preferred_approach"     name="preferred_approach"       type="STRING"/>
```

Comment:
Please avoid changing CSV structure. It is fine to add rows, but not columns. This file technically belongs to other project